scripts/core/ArticlesListByQueryWithFilters.tsx

```diff
@@ -192,9 +180,19 @@ export class ArticlesListByQueryWithFilters extends React.PureComponent<IProps,
                 },
             };
 
+            const item_types = [
+                {type: 'text', label: gettext('text')},
+                {type: 'picture', label: gettext('picture')},
+                {type: 'graphic', label: gettext('graphic')},
+                {type: 'composite', label: gettext('package')},
+                {type: 'highlight-pack', label: gettext('highlights package')},
+                {type: 'video', label: gettext('video')},
+                {type: 'audio', label: gettext('audio')},
+            ];
+
             const options: Array<IFileTypeOption> = [
                 filterAll,
-                ...getItemTypes().map((itemType) => {
+                ...getItemTypes(item_types).map((itemType) => {
```

Comment:
why do you pass those types?

Reply:
> why do you pass those types?

To remove duplicates I made a function **getItemTypes** in utils.ts which takes all available filters as an argument and returned required filters based on the **hideCreatePackage** setting.

Reply:
yep but seems that what you pass there is pretty much the same always, isn't it?

Reply:
No, Not exactly, Actually when it calls in `ArticlesListByQueryWithFilters.tsx` we did not pass **all filter** because it comes from the `filterAll` object. Therefore we need an argument to pass but I have made some changes and after that changes, we need not have to pass any argument.